bower.json

```diff
@@ -9,7 +9,7 @@
 	],
 	"dependencies": {
 		"o-colors": "^4.0.2",
-		"o-buttons": "^5.0.0",
+		"o-buttons": "^5.10.1",
```

Comment:
This could potentially be a reason for user to have bundle conflicts, not sure we worry about this though

Reply:
Good point. I didn't think about that. 

`5.10.1` would conflict due to a dependancy on a different major of `o-brand`. As `o-brand` is a direct dependancy only of origami components we thought we could release a new major seamlessly, but that assumes we can bump to a minor safety like this.

Think it's safe to keep as is?

Reply:
As discussed, going to leave as is. 🤞